[2024-05-08T21:57:07.779+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-08T21:57:07.806+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 10_branching.branch scheduled__2022-07-31T00:00:00+00:00 [queued]>
[2024-05-08T21:57:07.817+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 10_branching.branch scheduled__2022-07-31T00:00:00+00:00 [queued]>
[2024-05-08T21:57:07.818+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-05-08T21:57:07.835+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): branch> on 2022-07-31 00:00:00+00:00
[2024-05-08T21:57:07.842+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=4074) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-08T21:57:07.844+0000] {standard_task_runner.py:63} INFO - Started process 4080 to run task
[2024-05-08T21:57:07.843+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', '10_branching', 'branch', 'scheduled__2022-07-31T00:00:00+00:00', '--job-id', '908', '--raw', '--subdir', 'DAGS_FOLDER/10_branchpythonoperator.py', '--cfg-path', '/tmp/tmpuoebpucy']
[2024-05-08T21:57:07.845+0000] {standard_task_runner.py:91} INFO - Job 908: Subtask branch
[2024-05-08T21:57:07.890+0000] {task_command.py:426} INFO - Running <TaskInstance: 10_branching.branch scheduled__2022-07-31T00:00:00+00:00 [running]> on host ccc314eb571d
[2024-05-08T21:57:07.976+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='10_branching' AIRFLOW_CTX_TASK_ID='branch' AIRFLOW_CTX_EXECUTION_DATE='2022-07-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-07-31T00:00:00+00:00'
[2024-05-08T21:57:07.977+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-08T21:57:07.989+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-05-08T21:57:07.990+0000] {python.py:237} INFO - Done. Returned value was: start_15_june
[2024-05-08T21:57:07.991+0000] {branch.py:36} INFO - Branch into start_15_june
[2024-05-08T21:57:07.992+0000] {skipmixin.py:178} INFO - Following branch start_15_june
[2024-05-08T21:57:08.005+0000] {skipmixin.py:246} INFO - Skipping tasks [('finish_14_june', -1)]
[2024-05-08T21:57:08.027+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-08T21:57:08.045+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=10_branching, task_id=branch, run_id=scheduled__2022-07-31T00:00:00+00:00, execution_date=20220731T000000, start_date=20240508T215707, end_date=20240508T215708
[2024-05-08T21:57:08.098+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-08T21:57:08.122+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-08T21:57:08.126+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
